### PR TITLE
New task: Configure mail forwarging for root

### PR DESCRIPTION
**Pull request by cwickert**
[task-forward-root-mail_color_en.pdf](https://github.com/SUSE/doc-modular/files/7170328/task-forward-root-mail_color_en.pdf)
Feel free to make changes as you like. Please take care of merging this while I'm away over the weekend. TIA!

### CODE REVIEW

File: xml/task-forward-root-mail.xml [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+ <section xml:id="introduction-forward-root-mail">
+  <title>Introduction</title>
+  <para>
+   System daemons, <systemitem class="daemon">cron</systemitem> jobs, &systemd;
+   timers, and other applications can generate messages and send them to the
+   &rootuser; user of the system. By default, each user account owns a local
+   mailbox and will be notified about new mail messages upon login.
+  </para>
+  <para>
+   These messages can contain security relevant reports and incidents that might
```
Comment (lproven) [inline xml/task-forward-root-mail.xml:78]:
```suggestion
   These messages can contain security-relevant reports and incidents that might
```

File: xml/task-forward-root-mail.xml [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+      reason.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Was the test mail sent?</term>
+    <listitem>
+     <para>
+      Use the <command>mailq</command> command to verify that the test mail has
+      been sent. Upon success no mail should be pending in the queue.
```
Comment (lproven) [inline xml/task-forward-root-mail.xml:215]:
```suggestion
      been sent. Upon success, no mail should be pending in the queue.
```

File: xml/task-forward-root-mail.xml [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+      Verify you can ping the mail server with 
+      <command>ping -c 5 <replaceable>&mailname;</replaceable></command>
+      (where <replaceable>&mailname;</replaceable> is the host name of the mail
+      server returned by the <command>host</command> command in the previous
+      step.)
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Does the test mail still not arrive?</term>
```
Comment (lproven) [inline xml/task-forward-root-mail.xml:244]:
```suggestion
    <term>Is the test mail still not arriving?</term>
```

**Review by lproven — CHANGES_REQUESTED** [review]
Very minor suggestions only.

**Comment by cwickert** [comment]
I have applied all your suggestions. Thanks a lot!

**Review by lproven — APPROVED** [review]

**Pull request merged**